keycloak/data/havendev-realm.json

```diff
@@ -1,1598 +1,1778 @@
+        {
+          "id": "9984e791-591d-4262-82bd-e8b5723e2642",
+          "name": "manage-account-links",
+          "description": "${role_manage-account-links}",
+          "scopeParamRequired": false,
+          "composite": false,
+          "clientRole": true,
+          "containerId": "2a3f57d1-8bdb-43b3-ab45-2709c0214439"
+        }
+      ]
     }
   },
-  "groups" : [ {
-    "id" : "75bd19d2-f066-4357-881c-4ccb3d3a2cd0",
-    "name" : "admin",
-    "path" : "/admin",
-    "attributes" : { },
-    "realmRoles" : [ ],
-    "clientRoles" : { },
-    "subGroups" : [ ]
-  }, {
-    "id" : "8e1acd95-1ce2-4c16-963c-bbb1a25ca5ca",
-    "name" : "staff",
-    "path" : "/staff",
-    "attributes" : { },
-    "realmRoles" : [ ],
-    "clientRoles" : { },
-    "subGroups" : [ ]
-  }, {
-    "id" : "8f3b3184-d05e-477c-acd8-a44d46544b94",
-    "name" : "vendor",
-    "path" : "/vendor",
-    "attributes" : { },
-    "realmRoles" : [ ],
-    "clientRoles" : { },
-    "subGroups" : [ ]
-  } ],
-  "defaultRoles" : [ "uma_authorization", "member", "offline_access" ],
-  "requiredCredentials" : [ "password" ],
-  "passwordPolicy" : "hashIterations(20000)",
-  "otpPolicyType" : "totp",
-  "otpPolicyAlgorithm" : "HmacSHA1",
-  "otpPolicyInitialCounter" : 0,
-  "otpPolicyDigits" : 6,
-  "otpPolicyLookAheadWindow" : 1,
-  "otpPolicyPeriod" : 30,
-  "users" : [ {
-    "id" : "36c5fbbe-ff1b-4e8c-aad4-ca113bd70d37",
-    "createdTimestamp" : 1503871864986,
-    "username" : "elliot@kindlyops.com",
-    "enabled" : true,
-    "totp" : false,
-    "emailVerified" : true,
-    "firstName" : "Elliot",
-    "lastName" : "Murphy",
-    "email" : "elliot@kindlyops.com",
-    "attributes" : {
-      "role" : [ "admin" ]
+  "groups": [
+    {
+      "id": "75bd19d2-f066-4357-881c-4ccb3d3a2cd0",
+      "name": "admin",
+      "path": "/admin",
+      "attributes": {},
+      "realmRoles": [],
+      "clientRoles": {},
+      "subGroups": []
     },
-    "credentials" : [ {
-      "type" : "password",
-      "hashedSaltedValue" : "lQXRLpmv0E26q3Ds6zd/b6aIZ6bYbRqvfLjY88x7ITtfUCUJSwxatSfhPsKnFZ+sFR6ZGfJlNRsvbZnjP/UcOg==",
-      "salt" : "6u4mycWng8AhcPz2C+erhg==",
-      "hashIterations" : 20000,
-      "counter" : 0,
-      "algorithm" : "pbkdf2-sha256",
-      "digits" : 0,
-      "period" : 0,
-      "createdDate" : 1503871865370,
-      "config" : { }
-    } ],
-    "disableableCredentialTypes" : [ "password" ],
-    "requiredActions" : [ ],
-    "realmRoles" : [ "uma_authorization", "offline_access" ],
-    "clientRoles" : {
-      "account" : [ "manage-account", "view-profile" ]
+    {
+      "id": "63edc28f-95e6-4250-b784-67255c2d9001",
+      "name": "org-none",
+      "path": "/org-none",
+      "attributes": {},
+      "realmRoles": [],
+      "clientRoles": {},
+      "subGroups": []
     },
-    "groups" : [ ]
-  }, {
-    "id" : "90920d91-3090-4b4a-ae2a-2377cfa06ecd",
-    "createdTimestamp" : 1492023544974,
-    "username" : "user1@havengrc.com",
```

Comment:
it seems like this user definition (along with the other two users) got dropped in the export)